### PR TITLE
Another practice

### PR DESCRIPTION
**Pull request by slchow**
You can even add folders! But there's a lot of commits doing it this way.

### CODE REVIEW

**Comment by slchow** [comment]
@monicawamsley @danigsc can your review?

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# Training
\ No newline at end of file
+# Training
+
```
Comment (DaniGSC) [inline README.md:2]: Why si the top "training" line red?

Comment (slchow) [reply README.md:2]: Because it's been deleted

File: practice folder/practice.md
Comment (MonicaWamsley) [inline practice folder/practice.md:1]: Change "Training" to "Practice"

**Pull request merged**